docs/notifications.md

```diff
@@ -34,15 +34,15 @@ Filters use conjunctional logic.
    announcing the block itself
  * transaction notifications are only announced for successful transactions
  * all announcements are being done in the same order they happen on the chain
-   At first transaction execution is announced, then followed by notifications
-   generated during this execution, then followed by transaction announcement.
+   First, transaction execution is announced. Then, follow notifications
```

Comment:
The reader is not supposed to be following anything, instead notifications are following one another in the sequence of events.